RibonDAO/manager
# (TECH-65) Settings Details e Edit Page

# Description

- Settings page and edit settings page 

# Necessary changes

- Add page to settings details and edit
- Add tests

## Does this pull request close any open issues?

- closes [#TECH-65](https://ribon.atlassian.net/jira/software/c/projects/TECH/boards/26?modal=detail&selectedIssue=TECH-65)

## Type of change

Please delete options that are not relevant.


## How to test

- Select settings on menu and verify if details of settings and button edit. 
- Click in edit button and try to edit default ticket value.
